File: samples/nodejs/src/schemas/lead.js
const Joi = require('@hapi/joi');

const bodySchema = Joi.object({
  leadOrigin: Joi.string().required(),
  timestamp: Joi.string().required(),
  originLeadId: Joi.string().required(),
  originListingId: Joi.string().required(),
  clientListingId: Joi.string().required(),
  name: Joi.string().required(),
  email: Joi.string().required().regex(/^[\w+._%-]{1,256}@[\w][\w-]{0,64}(\.[\w][\w-]{0,25})+$/),
  ddd: Joi.string().required(),
  phone: Joi.string().required(),
  message: Joi.string().required(),
});

module.exports = {
  bodySchema,
};

File: samples/java/leadIntegration/src/main/java/br/com/example/leadIntegration/models/Lead.java
package br.com.example.leadIntegration.models;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Lead {
    private String leadOrigin;
    private String timestamp;
    private String originLeadId;
    private String originListingId;
    private String clientListingId;
    private String name;
    private String email;
    private String ddd;
    private String phone;
    private String message;
}